Documentation/Markdown/Events/ProvidedEvents/Form/DispatchFormNotification.md

```diff
@@ -0,0 +1,23 @@
+# Form finisher
+
+> /!\ For this event, you need to have [TYPO3 v8.5][form-changelog] minimum and the [Form][form-extension] extension installed and active.
+
+This event is triggered when a form that implements the notification dispatch
+finisher is submitted.
+
+The finisher class is `\CuyZ\Notiz\Domain\Event\Form\DispatchFormNotificationFinisher`.
```

Comment:
There is no need to indicate this.

However an explanation on how to configure the notification dispatch should be there.

You may use these images:
![image](https://user-images.githubusercontent.com/6342901/44724222-60e0cf00-aad2-11e8-862c-f7b952188e8f.png)

![image](https://user-images.githubusercontent.com/6342901/44724284-88d03280-aad2-11e8-844c-3a4a0e68f2f4.png)